app/src/main/res/values/strings.xml

```diff
@@ -613,5 +620,8 @@ Upload your first media by tapping on the add button.</string>
   <string name="app_ui_language">App user interface language</string>
   <string name="remove">Removes a caption and description</string>
   <string name="read_help_link">Read more</string>
+  <string name="access_token">
+    pk.eyJ1IjoiYXlhbi0xMCIsImEiOiJja293aGVucnAwMDZhMndwbm90cXV1MmR2In0.vRXyhsZPxsZNEyRU8gbDDw
```

Comment:
Is this really needed? Would it be made unnecessary by not using the Places plugin? Is it "legal" to publish it?
Please add an XML comment explaining this.